packages/babel-helper-define-map/src/index.js

```diff
@@ -65,8 +65,8 @@ export function push(mutatorMap: Object, node: Object, kind: string, file, scope
     value = nameFunction({ id: key, node: value, scope });
 
     // Class methods don't have their name bound in the funciton body.
-    if (t.isClassMethod(node)) {
-      value.id[t.NOT_LOCAL_BINDING] = true
+    if (t.isClassMethod(node) && value.id) {
```

Comment:
https://github.com/babel/babel/commit/95c93dd22b7d2475a7bc4f90554c96e7bd2ad204#commitcomment-15548270 mentions value is a `CallExpression` now after running `nameFunction` rather than a `FunctionExpression` like we expect.